third_party/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+# Suppress all warnings from third-party projects.
+set_property(DIRECTORY APPEND PROPERTY COMPILE_OPTIONS -w)
```

Comment:
Works on GCC and Clang. "[When compiling with MSVC the “-w” will be automatically converted to “/w” which has the same function.](https://www.johnlamp.net/cmake-tutorial-4-libraries-and-subdirectories.html)"